agentMET4FOF/ml_uncertainty/bnn.py

```diff
@@ -1,54 +1,11 @@
-from agentMET4FOF.agents import AgentMET4FOF
-from examples.ZEMA_EMC.zema_feature_extract import FFT_BFC, Pearson_FeatureSelection
-
-import numpy as np
-import time
-
-from matplotlib import pyplot as plt
-from sklearn.discriminant_analysis import LinearDiscriminantAnalysis
-from sklearn import linear_model
-from sklearn.ensemble import RandomForestRegressor
-from sklearn.metrics import mean_squared_error
-from sklearn.model_selection import train_test_split
-
-from sklearn.metrics import f1_score
-from sklearn.metrics import confusion_matrix
-
-from scipy.stats import skew
-from scipy.stats import kurtosis
-from scipy.fftpack import fft
-
-import pandas as pd
+from agentMET4FOF.ml_uncertainty.bnn import *
 
+from scipy import stats
+from scipy import stats
```

Comment:
This just is a nasty bug-side-effect.

Reply:
i just tried to pycharm optimiser (ctrl+alt +o) these days, i find that occassionally it is not that reliable though..

Reply:
Oh really? Could it be, that these cases occurred were as well in conjunction with * imports? I use this feature almost every day without any problem up until this one exceptional case. If there are problems with it or other cases where it is not advisable I would be glad if you could show or tell me.